Netezos.Tests/Forging/LocalForgingTest.cs

```diff
@@ -76,5 +77,70 @@ public async Task CompareToFile()
                 }
             }
         }
+
+        [Fact]
+        public async Task CompareUnforgedFromFile()
+        {
+            string opHex = @"../../../Forging/operations/ongeyECmEtrDFJbebtL1Frq4HPp369bvcTBZaL6kR4FsF36r72r/forged.hex";
+
+            byte[] localBytes = Hex.Parse(opHex);
+
+            byte[] operationBytes = localBytes.GetBytes(32, localBytes.Length - 32);
+
+            LocalForge localForge = new LocalForge();
+
+            //OriginationContent content = await localForge.UnforgeOperationAsync<OriginationContent>(operationBytes);
```

Comment:
This is preparation for doing a similar thing as the above method--reading all of the hex bytes from each operation, unforging them, and comparing them to the JSON from the unsigned file. I assume that will work (maybe with some massaging of JSON formatting or something).